### ISSUE TITLE
Projects section

### ISSUE THREAD

**Issue by armantorkzaban** [tcfev/Atlas#30]
A major problem in the Iranian diaspora is the lack of transparency about all the active projects with the focus of Iran, the initiatives behind them, amount of resources being invested, their methodologies, and evaluation metrics of their impact.

Here I suggest a section called Projects, where a search engine  and a list can give the user a look into (categories of) projects concluded or ongoing, who's behind which project, the promise of the project, their impact metrics, etc.

Some of the side products of this section would be gap analysis, activists finding and joining interesting projects, grand overview ...

@navidzand131 @next-free

**Comment by next-free** [comment]
i agree, there are more than 1000+ parties listed on activist resources but there is no any valuable data about what are they doing and what is (was) the achievements, the problem is finding data for this kind of analysis and creating valid information

**Comment by armantorkzaban** [comment]
This necessitates proper planning and resourcing. Let us set up a session for that.
